frontend/src/app/(routes)/multisig/Multisig.tsx

```diff
@@ -0,0 +1,8 @@
+'use client';
+import React from 'react';
+
+const Multisig = () => {
+  return <div>Multisig</div>;
```

Comment:
All multisig page is still pending?

Reply:
For multisig we don't have any overview page, we have chain specific multisig pages
/multisig/{chainame}